src/components/StatTile.vue

```diff
@@ -225,6 +225,11 @@ export default class StatTile extends Vue {
 
   // Returns a rounded number or undefined if no rank
   get propertyStatRank(): number | null {
+    if (["DistrictSteamUse", "DistrictChilledWaterUse"].includes(this.statKey))
```

Comment:
@SuragNuthulapaty - I'm going to fix this since you've done so much, but I wanted to flag up this is a brittle approach since it assumes that only two columns don't have a rank, when in reality only a subset do have a rank. In general I'd make sure that code is able to handle changes to interfaces that don't directly impact it, e.g. adding new fields to a function that only manipulates a specific set of fields